python/wrapper.cpp

```diff
@@ -106,6 +106,15 @@ void declare_float(nb::module_& m, std::string type)
              self.permute_subentity_closure(_d, entity_info, entity_type);
              return as_nbarray(std::move(_d));
            })
+      .def("permute_subentity_closure",
+           [](const FiniteElement<T>& self,
+              nb::ndarray<std::int32_t, nb::ndim<1>, nb::c_contig> d,
```

Comment:
Should this be const, i.e.
```cpp
nb::ndarray<const std::int32_t, nb::ndim<1>, nb::c_contig> d,
```

Reply:
No, it follows the same logic as the other permute functions, which does the permutation in-place.